packages/dm-core-plugins/src/attribute-selector/AttributeSelectorPlugin.tsx

```diff
@@ -31,14 +31,14 @@ export const AttributeSelectorPlugin = (
   const [views, setViews] = useState<TItemData[]>([])
   const [formData, setFormData] = useState<TGenericObject>({})
 
-  function addView(viewId: string, view: TViewConfig) {
+  function addView(viewId: string, view: TViewConfig, rootId?: string) {
```

Comment:
could not find anywhere in the changed code that addView() is used. Could you double check if the rootId is needed? :)

Reply:
Of course! addView is passed as the onOpen function in <Content />, so in both Table and List component, the onOpen function we call in openItemAsTab function is actually addView, and there we pass the rootId 🙂